test/controllers/concerns/current_user_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../../test_helper'
+
+SingleCov.covered! uncovered: 2
+
+describe "CurrentUser Controller" do
```

Comment:
worth being more obvious that this is testing a module (not an actual controller)?